File: oldEnough.py
print("This is a program that tells you if you can do certain things.")
age = int(input("How old are you? "))
if age >= 21:
    print("You are old enough to drink. \n")
elif age >= 18:
    print("You are old enough to vote. \n")
elif age >= 16:
    print("You are old enough to drive. \n")
elif age >= 15:
    print("You are old enough to get your learners permit. \n")
elif age >= 5:
    print("You are old enough to go to school. \n")
elif age < 5:
    print("You are not old enough to go to school. \n")

File: oldEnough.c
#include <stdio.h>

int main(void){
    int age;
    printf("How old are you?");
    scanf("%d", &age);
    if (age >= 21){
        printf("You are old enough to drink.");
    }else if (age >= 18){
        printf("You are old enough to vote");
    }else if (age >= 16){
        printf("You are old enough to drive.");
    }else if (age >= 15){
        printf("You are old enough to get a learners permit");
    }else if (age >= 5){
        printf("You are old enough to go to school.");
    }else if (age < 5);{
        printf("You are not old enough to go to school, can you even read this?");
    }
return 0;
}